rosetta-tools/src/main/java/com/regnosys/rosetta/tools/ResourceFormattingTool.java

```diff
@@ -39,42 +41,65 @@ public class ResourceFormattingTool {
 	private static Logger LOGGER = LoggerFactory.getLogger(ResourceFormattingTool.class);
 	
 	public static void main(String[] args) {
+		int maxArgs = 2;
+		
 		if (args.length == 0) {
-            System.out.println("Please provide the directory path as an argument.");
-            System.exit(1);
+            exitProgram("Please provide the directory path as an argument.");
+        }
+		
+		if (args.length > maxArgs) {
+			exitProgram("Too many arguments. Please provide maximum " + maxArgs + " arguments.");
         }
 		
 		Path directory = Paths.get(args[0]);
         if (!Files.isDirectory(directory)) {
-            System.out.println("The provided path is not a valid directory.");
-            System.exit(1);
+        	exitProgram("The provided path is not a valid directory.");
+        }
+        
+        // check if optional parameter was given. If not use default value
+        FormattingOptions formattingOptions = null;
+        if(args.length > 1) {
+        	String formattingOptionsPath = args[1];
+        	try {
+    			formattingOptions = ResourceFormatterMojo.readFormattingOptions(formattingOptionsPath);
+    		} catch (IOException e) {
+    			LOGGER.error("Config file not found.", e);
+    		} catch (MojoFailureException e) {
```

Comment:
Right now this is leaking Mojo specific exceptions. See previous comment to solve it.